src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java

```diff
@@ -18,102 +18,67 @@
 
 import com.amazonaws.AmazonServiceException;
 import com.amazonaws.services.cloudformation.model.StackStatus;
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 import com.nike.cerberus.command.core.UpdateStackCommand;
-import com.nike.cerberus.domain.cloudformation.CmsParameters;
-import com.nike.cerberus.domain.cloudformation.GatewayParameters;
-import com.nike.cerberus.domain.cloudformation.LaunchConfigParameters;
-import com.nike.cerberus.domain.environment.Stack;
+import com.nike.cerberus.domain.EnvironmentMetadata;
 import com.nike.cerberus.operation.Operation;
 import com.nike.cerberus.operation.UnexpectedCloudFormationStatusException;
-import com.nike.cerberus.service.AmiTagCheckService;
 import com.nike.cerberus.service.CloudFormationService;
 import com.nike.cerberus.service.Ec2UserDataService;
-import com.nike.cerberus.store.ConfigStore;
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.inject.Inject;
-import javax.inject.Named;
-import java.util.HashMap;
 import java.util.Map;
 
 import static com.amazonaws.services.cloudformation.model.StackStatus.UPDATE_COMPLETE;
 import static com.amazonaws.services.cloudformation.model.StackStatus.UPDATE_COMPLETE_CLEANUP_IN_PROGRESS;
 import static com.amazonaws.services.cloudformation.model.StackStatus.UPDATE_ROLLBACK_COMPLETE;
 import static com.amazonaws.services.cloudformation.model.StackStatus.UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS;
 import static com.amazonaws.services.cloudformation.model.StackStatus.UPDATE_ROLLBACK_FAILED;
-import static com.nike.cerberus.module.CerberusModule.CF_OBJECT_MAPPER;
 
 /**
  * Operation for updating stacks.
  */
 public class UpdateStackOperation implements Operation<UpdateStackCommand> {
 
-    private final Logger logger = LoggerFactory.getLogger(getClass());
-
-    private final Map<Stack, Class<? extends LaunchConfigParameters>> stackParameterMap;
 
-    private final ConfigStore configStore;
```

Comment:
The configStore variable here reminds me that certain stacks will require updates to config in S3.

Maybe each stack command should implement it's own update method that the UpdateStackOperation will call. Or we could punt on that and add a `requiresConfigUpdate` on the `Stack` object so that the UpdateStackOperation does not allow users to update stacks without also updating the necessary values in S3